chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -263,7 +263,22 @@ BravePrepopulatedEngineID GetDefaultSearchEngine(int country_id, int version) {
           {country_codes::CountryCharsToCountryID('U', 'Z'),
            PREPOPULATED_ENGINE_ID_YANDEX},
       });
-
+  static const base::NoDestructor<
+      base::flat_map<int, BravePrepopulatedEngineID>>
+      content_v20({
+          {country_codes::CountryCharsToCountryID('A', 'T'),
+           PREPOPULATED_ENGINE_ID_BRAVE},
+          {country_codes::CountryCharsToCountryID('E', 'S'),
+           PREPOPULATED_ENGINE_ID_BRAVE},
+          {country_codes::CountryCharsToCountryID('M', 'X'),
+           PREPOPULATED_ENGINE_ID_BRAVE},
+      });
+  if (version > 19) {
+    auto it = content_v20->find(country_id);
+    if (it != content_v20->end()) {
+      return it->second;
+    }
```

Comment:
Can the above match the below style for checks?
```
    auto it = content_v20->find(country_id);
    if (it == content_v20->end()) {
      return default_v6;
    }
    return it->second;
```

Basically, we want to enforce that if the default is not found in this list, we should return the last global default (which happens to be Google in this case)

Reply:
done